.github/workflows/integration.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v4
       with: 
-        go-version: '1.20.7'
+        go-version: '1.21.1'
```

Comment:
Can you double check upstream? I think it's 1.20.8.

Reply:
Yes, the `GOVERSION` variable was set before switching to the release branch